src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XslCompiledTransform.cs

```diff
@@ -1395,6 +1395,7 @@ public void LoadGeneric10()
         */
 
         //[Variation(id = 11, Desc = "Load stylesheet with entity reference: Bug #68450 ")]
+        [ActiveIssue(9877)]
```

Comment:
These tests are using checksum to verify the results, which can be flaky and turns out to fail all the time on Linux. There is [an issue](https://github.com/dotnet/corefx/issues/9877) tracking this to rewrite these tests to not use checksum anymore.